setup.py

```diff
@@ -33,6 +33,7 @@
         'pescador >= 0.1.2',
         'pytest',
         'pytest-benchmark',
+        'pandas >= 0.17.0'
```

Comment:
http://www.wcvb.com/national/aww-giant-panda-takes-a-tumble-in-the-snow/38121224